src/packages/@ncigdc/dux/bannerNotification.js

```diff
@@ -33,7 +33,7 @@ let initialState = [
   {
     components: ['PORTAL'],
     level: 'INFO',
-    id: 0,
+    id: 'initial_banner',
     dismissible: true,
     message: <DismissibleBanner />,
```

Comment:
it's for setting that `Can't find your data?` dismissible notification that's hardcoded and not coming from the endpoint